### ISSUE TITLE
Source Static files from the Project

### ISSUE THREAD

**Issue by satish860** [DamianEdwards/vNextBowerGrunt#1]
How we can structure our project for the source code. I am taking here about source files which needs to need transposed (Coffeescript,ES6,Typescript) . 
1) Can we keep it outside and build it to wwwroot .
2) If we have HTML file where it will be . We use Index.html instead of Razor in our project.

**Comment by davidfowl** [comment]
> 1) Can we keep it outside and build it to wwwroot .

Yes, things in the `wwwroot` are the things that the web server needs to see.

> 2) If we have HTML file where it will be . We use Index.html instead of Razor in our project.

Index.html needs to go into the `wwwroot` because the web server needs to see it.
